manual/manual_backend.rst

```diff
@@ -53,18 +53,19 @@ Knowing the requirements of each :py:class:`~pytket.backends.Backend` is handy i
 
 .. jupyter-execute::
 
-    from pytket import Circuit
+    from pytket import Circuit, OpType
     from pytket.extensions.qiskit import AerBackend
 
-    circ = Circuit(2, 2)
-    circ.Rx(0.3, 0).Ry(0.5, 1).CRz(-0.6, 1, 0).measure_all()
+    circ = Circuit(3, 2)
+    circ.H(0).Ry(0.25, 1)
+    circ.add_gate(OpType.CnRy, [0.74], [0, 1, 2]) # CnRy not in AerBackend gate set
+    circ.measure_all()
 
     backend = AerBackend()
     if not backend.valid_circuit(circ):
-        compiled_circ = backend.get_compiled_circuit(circ)
-        assert backend.valid_circuit(compiled_circ)
+        compiled_circ = backend.get_compiled_circuit(circ) # Compile circuit to AerBackend
 
-    print(compiled_circ.get_commands())
+    print("Valid circuit for AerBackend?", backend.valid_circuit(compiled_circ))
```

Comment:
This woud still still fail if the original circuit was valid because `compiled_circ` would not exist. Even though we don't expect this to happen it would be better to avoid the possibility. There are several ways to go about it, but maybe the simplest is to just indent line 68.

Reply:
In fact I've just removed the if statement as I think its confusing here.